dotnet/DataSources/IProductReviewRepository.cs

```diff
@@ -25,7 +25,7 @@ public interface IProductReviewRepository
         Task<string> VerifySchema();
         Task<string> VerifyMigration();
         Task<string> SuccessfulMigration();
-        Task<ReviewsResponseWrapper> GetProductReviewsMD(string searchQuery, string from = null, string to = null);
+        Task<ReviewsResponseWrapper> GetProductReviewsMD(string searchQuery, string from, string to);
```

Comment:
Setting these parameters to null makes them optional.  That way we can avoid having to set them to null in the cases where the pagination is not specified.

Reply:
Thank you for the information